near-sdk/Cargo.toml

```diff
@@ -61,6 +61,11 @@ rand_chacha = "0.3.1"
 near-rng = "0.1.1"
 near-abi = { version = "0.4.0", features = ["__chunked-entries"] }
 symbolic-debuginfo = "12"
+near-workspaces = { git = "https://github.com/near/near-workspaces-rs", branch = "main"}
```

Comment:
@frol not the git dependency here. It would actually be nice to release workspaces and fix this.